config/settings.yml

```diff
@@ -107,6 +107,8 @@ attachments_dir: ":rails_root/public/attachments"
 #logstash_path: ""
 # List of memcache servers to use for caching
 #memcache_servers: []
+# URL of Maptiler API for vector maps
+maptiler_url: "https://api.maptiler.com/"
```

Comment:
Does this actually need to be a setting? The other layers are all hard coded and in fact this is hard coded in the javascript, just not when setting the security policy.

What we should have in here is a commented out setting for `maptiler_key` though, as we already do for the other map layers that need a key.